src/Domain/EmbeddingMaps/AffineMap.hpp

```diff
@@ -14,19 +14,27 @@
 
 namespace EmbeddingMaps {
```

Comment:
Yes, though I the general advice for constructors is to avoid inlining them unless measurements show it's faster. Since we shouldn't be constructing embedding maps that often, and the heap allocation for the `unique_ptr` that will almost always hold them is very likely to dominate the time I think it's better to focus on reducing compile time. Similarly, for the pup functioning, we shouldn't be serializing embedding maps that often, and if we are we have to address that issue first. Sound okay?